quickstart-node/README.md

```diff
@@ -1,11 +1,13 @@
 # Monday-Code NodeJS Quickstart
 
-This Node - express with typescript application is designed to showcase the practical use of the monday code platform by integrating various monday app functionalities.
-It's an essential resource for developers looking to leverage Monday Code features through NodeJS environment.
+This Node - express with typescript application is designed to showcase the practical use of monday code and monday api
+sdk by integrating various monday app functionalities. It's an essential resource for developers looking to leverage
+Monday Code and Monday API features through NodeJS environment.
 
 ## Getting Started
```

Comment:
Sorry one more thing – IMO you should use the monday apps tunnel in your `npm run dev` script... 

Check our other quickstarts for an example - https://github.com/mondaycom/welcome-apps/blob/2015c76296c6ecc36b7dbaa6559adc4758342757/apps/quickstart-integrations/package.json#L7